src/main/java/net/dv8tion/jda/api/interactions/commands/build/OptionData.java

```diff
@@ -316,19 +316,21 @@ public OptionData addChoice(@Nonnull String name, double value)
      *         <ul>
      *             <li>{@code name} is not null, empty and less or equal to {@value #MAX_CHOICE_NAME_LENGTH} characters long</li>
      *             <li>The amount of already set choices is less than {@link #MAX_CHOICES}</li>
-     *             <li>The {@link OptionType} is {@link OptionType#INTEGER}</li>
+     *             <li>The {@link OptionType} is not {@link OptionType#INTEGER}</li>
      *         </ul>
      *
      * @return The OptionData instance, for chaining
      */
     @Nonnull
-    public OptionData addChoice(@Nonnull String name, int value)
+    public OptionData addChoice(@Nonnull String name, long value)
```

Comment:
This is a breaking change to the method signature.

Reply:
I don't think it is since every int fits in a long anyway. How else should i do it?